core/db_classes/EE_Event.class.php

```diff
@@ -1427,20 +1427,20 @@ public function registrationFormUuid():? string
     public function registrationForm()
     {
         $FSC_UUID = $this->registrationFormUuid();
-        return !empty($FSC_UUID)
-            ? $this->get_many_related(
-                'Form_Section',
-                [
-                    [
-                        'OR' => [
-                            'Form_Section.FSC_UUID'      => $FSC_UUID, // top level form
-                            'Form_Section.FSC_belongsTo' => $FSC_UUID, // child form sections
-                        ]
-                    ],
-                    'order_by' => ['FSC_order' => 'ASC'],
+
+        if (empty($FSC_UUID)) {
+            return [];
+        }
+        
+        return EEM_Form_Section::instance()->get_all([
+            [
+                'OR' => [
+                    'FSC_UUID'      => $FSC_UUID, // top level form
+                    'FSC_belongsTo' => $FSC_UUID, // child form sections
                 ]
-            )
-            : [];
+                ],
+            'order_by' => ['FSC_order' => 'ASC'],
+        ]);
     }
```

Comment:
@tn3rb the reason for this change is that `->get_many_related()` adds the relation field with `AND` clause to the query thus invalidating the query for sections, thus 
```php
$this->get_many_related(
    'Form_Section',
    [
        [
            'OR' => [
                'Form_Section.FSC_UUID'      => $FSC_UUID, // top level form
                'Form_Section.FSC_belongsTo' => $FSC_UUID, // child form sections
            ]
        ]
  ]
);
```
is equivalent to 
```php
$this->get_many_related(
    'Form_Section',
    [
        'AND' => [
            'OR' => [
                'Form_Section.FSC_UUID'      => $FSC_UUID, // top level form
                'Form_Section.FSC_belongsTo' => $FSC_UUID, // child form sections
            ]
            'Form_Section.FSC_UUID'      => $FSC_UUID, // top level form
        ]
    ]
);
```

Reply:
oh wow... hmmm.. ok we'll go with your fix for now, but I think i'll create an issue to fix that in the models